docs/docs/events/builtin/extra/security_bpf_prog.md

```diff
@@ -0,0 +1,51 @@
+# security_bpf_prog
+
+## Intro
+security_bpf_prog - Do a check when the kernel generate and return a file descriptor for BPF programs.
+
+## Description
+This event marks the act of getting a file descriptor of a BPF program. It is triggered when the BPF program is being 
+loaded, or when the user asks for it explicitly. This event is of type LSM hook.
+The event contains data about the BPF program, and whether it is currently being loaded or not. 
+
+
+## Arguments
+* `type`:`int`[K] - the BPF program type.
+* `name`:`const char*`[K] - the BPF program name (first 16 bytes only, as this is how it is saved in the kernel).
+* `helpers`:`unsigned long[]`[K] - list of all BPF helpers being used by the BPF program.
+* `id`:`u32`[K] - the BPF program ID as set by the kernel.
+* `load`:`bool`[K] - whether this BPF program is currently being loaded.
+
+## Hooks
+### security_bpf_prog
+#### Type
+kprobe
+#### Purpose
+The LSM hook of getting a file descriptor of a BPF program. This hook triggers the event. 
+
+### bpf_check
+#### Type
+kprobe
+#### Purpose
+Save data of whether this BPF program is currently being loaded.
+
+### check_helper_call
+#### Type
+kprobe
+#### Purpose
+check whether the BPF program uses helper functions of interest
+
+### check_map_func_compatibility
+#### Type
+kprobe
+#### Purpose
+check whether the BPF program uses helper functions of interest
+
+## Example Use Case
+./tracee -f e=security_bpf_prog
+
+## Issues
+The 'check_helper_call' and 'check_map_func_compatibility' serves the same purpose. 
```

Comment:
This isn't true anymore. Now we have the `SetLibbpfgoCallbacks()` and `check_map_func_compatibility()` error isn't displayed. You should check if the `check_helper_call` is displayed and filter it out. We're expecting those errors so we should filter them out for now (until we can probe for the kprobes existence with feature probing feature.

The code is at:

https://github.com/aquasecurity/tracee/blob/c9ec790e1ddd43853531fa5fdee1e316238495cf/pkg/cmd/initialize/callbacks.go#L17-L24

Reply:
i went over the output of all the test envs, and no warning/error was generated in regards to these probes. so we're good to remove this comment.
i'll update it now